### DISCUSSION TITLE
change offline map url and voice

### DISCUSSION THREAD

**Discussion by akbarazimifar** [osmandapp/OsmAnd#15658] [Q&A]
Hello . How to customize the program , change the default language of the program and the default sound and address to receive the sound and map? Please help me . Thanks

**Comment by sonora** [comment]
- In the app: Tap the menu button in the lower left corner of the map screen, scroll the menu up a bit and select "Help".
- On the web: https://osmand.net/docs/user/

**Comment by akbarazimifar** [comment]
> 

I want to make these changes for the user and it has been changed by default. Not for the user to change the settings.

**Comment by akbarazimifar** [comment]
bye See, for example, I plan to change these default options for my users. Before making apk. Please help me . See the image below.


![7bea845631aaf005dede2ff9c1db879f](https://user-images.githubusercontent.com/5562558/200105829-e3ce6677-63b6-4a71-92e9-270bdafd5b96.jpg)

![185662f02a6181f110ddc467d2e70997](https://user-images.githubusercontent.com/5562558/200105846-b09b4037-f840-4e07-97c6-341ee368ee89.jpg)
